enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/PredefinedRolesBuilder.java

```diff
@@ -46,6 +47,7 @@
         SimpleRole architect = new SimpleRole( ARCHITECT );
         architect.add( new WildcardPermission( "schema:*" ) );
         architect.add( new WildcardPermission( "data:*" ) );
+        architect.add( new WildcardPermission( "token:*" ) );
```

Comment:
These tokens could be constants. Would slightly speed up the Set operations inside `SimpleRole` when things are added and deleted.